packages/core/src/Modules/Endpoint/Endpoint.jsx

```diff
@@ -14,10 +14,12 @@ import { observer, useStore } from '@deriv/stores';
 
 const FeatureFlagsSection = observer(() => {
     const { feature_flags } = useStore();
+    const HIDDEN_FEATURE_FLAGS = ['wallet'];
```

Comment:
aren't developers still using it to access v1 features when porting them to v2?
also, why hiding? if its genuinely not needed, would it be better to remove it?

Reply:
resolved as per chat: https://deriv-group.slack.com/archives/C0548T15K1P/p1698321780733129?thread_ts=1698289391.996029&cid=C0548T15K1P